pkgs/development/libraries/libinih/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, meson, ninja }:
+
+stdenv.mkDerivation rec {
+  pname = "libinih";
+  version = "52";
+
+  src = fetchFromGitHub {
+    owner = "benhoyt";
+    repo = "inih";
+    rev = "r${version}";
+    sha256 = "0lsvm34zabvi1xlximybzvgc58zb90mm3b9babwxlqs05jy871m4";
+  };
+
+  mesonFlags = [
+    "-Ddefault_library=shared"
+    "-Ddistro_install=true"
+  ];
+
+  nativeBuildInputs = [
+    meson
+    ninja
+  ];
+
+  meta = with stdenv.lib; {
+    homepage = "";
+    description = "A simple .INI file parser written in C";
+    license = licenses.bsd3;
+    platforms = platforms.linux;
```

Comment:
Why `platforms.linux` instead of `platforms.all`?

Reply:
Because I can't test anything else and I therefore cannot support them as a maintainer.